core/dev-tools/templates/simple/microfrontend.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<title></title>
+	<meta charset="utf-8">
+  <link href="./public/luigi.css" rel="stylesheet">

```

Comment:
It would work here, but just for consistency.
```suggestion
  <link href="/public/luigi.css" rel="stylesheet">

```